tools/install-tcmalloc.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+#
+# Script to install libtcmalloc.so in the current WiredTiger git
+# repository.
+#
+# This is serves the same purpose, but is NOT THE SAME as:
+#
+# test/evergreen/tcmalloc_install_or_build.sh
+#
+# and the tcmalloc source version referenced here should be updated in
+# sync with that script.
```

Comment:
Can we have the other script refer to this script as well? Or have the tcmalloc source version (lines #22--24, #41--44 below) abstracted into a common file that both scripts can source (to avoid potential version upgrade inconsistency)

Reply:
There isn't a logical place afaict for this given the different directories they live are at the top-level.
This is a rarely used script: so maybe it is better to require an argument that requires the user grab the relevant information from the script under `test/`.

Reply:
> maybe it is better to require an argument that requires the user grab the relevant information from the script under test/?

Sounds good. As long as users get some reminder about the tcmalloc version that should be installed (the version that's actively tested in Evergreen) while running this script.